fields/update_test.go

```diff
@@ -39,6 +39,10 @@ func TestUpdateBasic(t *testing.T) {
 		{"set_nested", Set("a.b.c", 123), `{"$set":{"a.b.c":123}}`, nil},
 		{"unset_nested", Unset("a.b.c"), `{"$unset":{"a.b.c":null}}`, nil},
 		{"set.unset_duplicate", Set("a1", 123).Unset("b1").Set("a1", 123).Unset("b1"), `{"$set":{"a1":123},"$unset":{"b1":null}}`, nil},
+		{"increment", Increment("a", 123), `{"$increment":{"a":123}}`, nil},
+		{"decrement", Decrement("a", 123), `{"$decrement":{"a":123}}`, nil},
+		{"multiply", Multiply("a", 123), `{"$multiply":{"a":123}}`, nil},
+		{"divide", Divide("a", 123), `{"$divide":{"a":123}}`, nil},
```

Comment:
The diff looks good overall, could add one more test here, which would test chained calls.
Like:
```
Set("a1", 123).Increment("a", 1231).Decrement("a", 1232).Multiply("a", 1233).Divide("a", 1234)
```